dialects/mysql/src/main/kotlin/app/cash/sqldelight/dialects/mysql/MySqlTypeResolver.kt

```diff
@@ -136,3 +175,7 @@ class MySqlTypeResolver(
     )
   }
 }
+
+private fun PsiElement.childOfType(types: TokenSet): PsiElement? {
```

Comment:
This is copied from `:sqldelight-compiler`. I could add a dep instead but I'm not sure thats desirable.

Reply:
Another option: Move it from compiler into dialect. But I am fine with this copy here.